UNRELEASED.md

```diff
@@ -4,6 +4,10 @@
 
 ### Enhancements
 
+- Updated `Popover` to focus the correct element when closed ([#2255](https://github.com/Shopify/polaris-react/pull/2255))
```

Comment:
You've got some stuff here that shouldn't be. These lines were probably included in a release but and got mixed up in a rebase or something.

Reply:
This has been fixed now I think 💯